src/orga/getting-in-touch.rst

```diff
@@ -0,0 +1,25 @@
+Getting in Touch
+================
+
+There are many ways to get in touch with the conda-forge community, primarily through issue trackers, mailing lists and real-time chat.
+
+The main issue trackers that you will interact with are
+* [staged-recipes](https://github.com/conda-forge/staged-recipes/issues). You'll use staged-recipes to create a new conda package on conda-forge
+* [our main docs repo](https://github.com/conda-forge/conda-forge.github.io/issues). You'll use this repo as a catch-all for issues where you're not sure where else to put them
+* [our enhancement proposals repo](https://github.com/conda-forge/cfep/issues). You'll use the enhancement proposals repo if you're interested in substantially changing the way conda-forge operates.
+
+The main chat rooms that you'll interact with are:
```

Comment:
bot gitter room?

Reply:
Can you tell me the room name?

Reply:
yup. 

conda-forge/regro-cf-autotick-bot